docs/source/usersguide/input.rst

```diff
@@ -1475,114 +1475,233 @@ The ``<tally>`` element accepts the following sub-elements:
-      Legendre expansion order of the change in particle angle
-      :math:`\left(\mu\right)`. That is, ``scatter-P1`` is equivalent to
-      requesting tallies of ``scatter-0`` and ``scatter-1``.  Like for
-      ``scatter-N``, N must be between 0 and 10. As an example, tallying up to
-      the 2\ :sup:`nd` \ scattering moment would be specified as ``<scores>
-      scatter-P2 </scores>``. Units are reactions per source particle.
-
-    :scatter-YN:
-      ``scatter-YN`` is similar to ``scatter-PN`` except an additional expansion
-      is performed for the incoming particle direction
-      :math:`\left(\Omega\right)` using the real spherical harmonics.  This is
-      useful for performing angular flux moment weighting of the scattering
-      moments. Like ``scatter-PN``, ``scatter-YN`` will tally all of the moments
-      from order 0 to N; N again must be between 0 and 10. Units are reactions
-      per source particle.
-
-    :nu-scatter, nu-scatter-N, nu-scatter-PN, nu-scatter-YN:
-      These scores are similar in functionality to their ``scatter*``
-      equivalents except the total production of neutrons due to scattering is
-      scored vice simply the scattering rate. This accounts for multiplicity
-      from (n,2n), (n,3n), and (n,4n) reactions. Units are neutrons produced per
-      source particle.
-
-    :flux-YN:
-      Spherical harmonic expansion of the direction of motion
-      :math:`\left(\Omega\right)` of the total flux.  This score will tally all
-      of the harmonic moments of order 0 to N.  N must be between 0
-      and 10. Units are particle-cm per source particle.
-
-    :total-YN:
-      The total reaction rate expanded via spherical harmonics about the
-      direction of motion of the neutron, :math:`\Omega`.
-      This score will tally all of the harmonic moments of order 0 to N.  N must
-      be between 0 and 10. Units are reactions per source particle.
-
-    :current:
-      Partial currents on the boundaries of each cell in a mesh. Units are
-      particles per source particle.
-
-      .. note::
-          This score can only be used if a mesh filter has been
-          specified. Furthermore, it may not be used in conjunction with any
-          other score.
-
-    :inverse-velocity:
-      The flux-weighted inverse velocity where the velocity is in units of
-      centimeters per second.
-
-      .. note::
-         The ``analog`` estimator is actually identical to the ``collision``
-         estimator for the inverse-velocity score.
-
-    :events:
-      Number of scoring events. Units are events per source particle.
+    A space-separated list of the desired responses to be accumulated. The accepted
+    options are listed in the following table:
+
+    .. table:: Score types available in OpenMC
+
+        +----------------------+---------------------------------------------------+
+        |Score                 | Description                                       |
```

Comment:
Two thoughts on this table:

* perhaps put the scores in double quotations to mirror their appearance in either XML or Python
* perhaps alphabetically order the scores?

I realize that these are separate from the motivation behind this PR, but are two things that come to mind when reading this table over.